src/backend/src/routes/work-packages.routes.ts

```diff
@@ -15,6 +15,7 @@ workPackagesRouter.post(
   intMinZero(body('projectWbsNum.carNumber')),
   intMinZero(body('projectWbsNum.projectNumber')),
   intMinZero(body('projectWbsNum.workPackageNumber')),
+  body('stage'),
```

Comment:
validation?

Reply:
what would we validate it as? String or null?

Reply:
that it's a valid value in the enum but also optional, look around the other files

Reply:
Do we want to send null from the frontend? Its causing like weird bugs. I dont have the most experience with sending optionals and these weird select control vs uncontrolled components